src/Faithlife.Tracing/AspNet/AspNetMvcTracing.cs

```diff
@@ -0,0 +1,18 @@
+#if NET461
+using System.Web;
+using System.Web.Mvc;
+
+namespace Faithlife.Tracing.AspNet
+{
+	public static class AspNetMvcTracing
```

Comment:
I might put this in its own `Faithlife.Tracing.AspNetMvc` namespace/assembly.

Web API would go in `Faithlife.Tracing.AspNetWebApi`.

Maybe that's overkill; maybe they share more types than I think.

Is ASP.NET Core separate, too?

Reply:
> Maybe that's overkill; maybe they share more types than I think.

They don't share public types (well, except for `AspNetTracingSettings`). The implementation could be shared behind-the-scenes.

> Is ASP.NET Core separate, too?

Most likely, but I haven't built it yet.